app/views/users/index.haml

```diff
@@ -1,7 +1,7 @@
 - if @inactive
-  %h1 Manage Inactive Drivers
+  %h1 Inactive Drivers
```

Comment:
Not for accessibility, I just thought it was confusing to have an h1 saying 'Manage Inactive Drivers' immediately followed by a link reading 'Manage Active Drivers', for example. So I took the imperative verb out of the title.